test/test_regex.py

```diff
@@ -1,9 +1,15 @@
+"""
+Unittests of asm2cfg's regexes
+"""
+
 import unittest
 
 from src.asm2cfg import asm2cfg
 
 
 class FunctionHeaderTestCase(unittest.TestCase):
+    "Tests of function header regex"
```

Comment:
Same here. Let's use consistent style of comments here and in other functions in this module.